bedrock/settings/__init__.py

```diff
@@ -111,7 +111,7 @@
         # support older browsers (mainly Safari)
         "frame-src": _csp_child_src,
         "frame-ancestors": [csp.constants.NONE],
-        "upgrade-insecure-requests": True,
+        "upgrade-insecure-requests": False if DEBUG else True,
```

Comment:
```suggestion
        "upgrade-insecure-requests": False if DEV and DEBUG else True,  # https://webkit.org/b/250776
```

I know this might be a bit overboard, but… a little more explicit to make it extra bulletproof just in case?

Reply:
I'm curious why the extra check for `DEV`?

`DEV` is true on the demos but they are using https. I feel like `DEBUG` is the master switch here but I could be missing something.

Reply:
Oh I just thought that both should be passing at once, for extra measure. I'm trying to find some similar patterns, as there's a little bit of overlap as some of the deployments are both DEV and PROD at the same time(?)… so I probably just meant:

```suggestion
        "upgrade-insecure-requests": False if not PROD and DEBUG else True,  # https://webkit.org/b/250776
```

But still looking, e.g. Markus/datadog seems to be disabled by DEBUG, as is USE_ETAGS etc. — can't tell if that is all similar cases, meaning "extra tweaks for local work", but sounds like it, can't really think of any non-PROD deployment where DEBUG would make sense, so that is being relied upon as a sort of local-dev-only-equivalent flag in a broader sense perhaps?

Reply:
`PROD` is more of the "Is this a production environment?" config.
`DEV` I think of as a "mode". In dev mode all switches are active by default and all locales are enabled, things like that.

I'll merge this as-is, based on `DEBUG`, so at least local dev isn't broken.